.github/workflows/gradle.yml

```diff
@@ -5,9 +5,9 @@ name: Java CI with Gradle
 
 on:
   push:
-    branches: [ master ]
+    branches: [ master, dev, puzzle-editor ]
```

Comment:
We could probably just run this on every branch push instead. Puzzle-editor might not always exist and the more checks the better.

Reply:
Yeah ok honestly sure why not. Let's change it to all branches.